dev/build.image/profiles/microProfile4/features.xml

```diff
@@ -1,3 +1,19 @@
 <feature>microProfile-4.0</feature>
 <feature>microProfile-4.1</feature>
 <feature>localConnector-1.0</feature>
+<feature>appSecurity</feature>
+<feature>cdi</feature>
+<feature>el</feature>
```

Comment:
```suggestion
<feature>cdi</feature>
```
I would prefer not including the features that are not required by MP.  They will be brought in by the other features.